pages/products/index.js

```diff
@@ -0,0 +1,122 @@
+import Link from 'next/link';
+import Head from 'next/head';
+import Header from '../../components/Header.js';
+import Footer from '../../components/Footer.js';
+import React, { useState } from 'react';
+import SearchBar from '../../components/SearchBar.js';
+
+export default function Products({ products }) {
+  const [search, setSearch] = useState('');
+  const [filter, setFilter] = useState('not active');
+
+  function searchFunction(e) {
+    setSearch(e.target.value);
+    setFilter('active');
+  }
+
+  return (
+    <div>
+      <Head>
+        <link rel="icon" href="/favicon.png" />
+        <link
+          href="https://fonts.googleapis.com/css2?family=DM+Mono&display=swap"
+          rel="stylesheet"
+        ></link>
+        <title>Bist du Teppich! Shop</title>
+      </Head>
+      ;
+      <Header />
+      <div className="rugtainer">
+        <SearchBar products={products} searchFunction={searchFunction} />
+        {products
+          .filter((item) => {
+            if (filter === 'active') {
```

Comment:
Instead of the separate second state variable (`filter`), you could just check if `search` is not `''` 🤓